coreapi/v23.0/keymanager/api/secret.go

```diff
@@ -31,6 +34,21 @@ type EncryptedSecret struct {
 	Ciphertexts map[x25519.PublicKey][]byte `json:"ciphertexts"`
 }
 
+// We implement MarshalJSON here because the the encoding/json library
```

Comment:
"We" is ambiguous

```suggestion
// XXX: Nexus-specific addition/hack.
// We implement MarshalJSON here because the the encoding/json library
```